packages/shortest/src/browser/core/browser-tool.ts

```diff
@@ -313,7 +362,31 @@ export class BrowserTool extends BaseBrowserTool {
           break;
         }
 
-        case "github_login": {
+        case InternalActionEnum.HOLD_KEY: {
+          if (!input.text) {
+            throw new ToolError("Key required for hold_key action");
+          }
+
+          if (!input.duration) {
+            throw new ToolError("Duration required for hold_key action");
+          }
+
+          const seconds = input.duration;
+          const delay = seconds / 1000;
+
+          const keyText = input.text.toLowerCase();
+          const keys = Array.isArray(actions.keyboardShortcuts[keyText])
+            ? actions.keyboardShortcuts[keyText]
+            : [actions.keyboardShortcuts[keyText] || input.text];
```

Comment:
Kept similar logic to `key` action. Generally, I think this is not needed as much, as Playwright accepts a combo key `(e.g. `Ctrl+c`). Leaving for now, can be removed in the future.